### PR TITLE
Fix(curriculum): external RMS link in CIP

### PR DESCRIPTION
**Pull request by Sboonny**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

128 in #40749

### CODE REVIEW

File: curriculum/challenges/english/10-coding-interview-prep/rosetta-code/averages-root-mean-square.md [hunk 1]
```diff
@@ -8,11 +8,9 @@ dashedName: averagesroot-mean-square
 
 # --description--
 
-Compute the [Root mean square](<https://en.wikipedia.org/wiki/Root mean square> "wp: Root mean square") of the numbers 1 through 10 inclusive.
+Compute the Root mean square of the numbers 1 through 10 inclusive.
 
-The *root mean square* is also known by its initials RMS (or rms), and as the **quadratic mean**.
-
-The RMS is calculated as the mean of the squares of the numbers, square-rooted:
+The <abbr title="Root mean square">RMS</abbr> is calculated as the mean of the squares of the numbers, square-rooted:
```
Comment (Sembauke) [inline curriculum/challenges/english/10-coding-interview-prep/rosetta-code/averages-root-mean-square.md:13]:
```suggestion
The <abbr title="Root mean square">RMS</abbr> is calculated by taking the mean of the squares of the numbers, and then square-rooting that number:
```

**Review by Sembauke — DISMISSED** [review]

File: curriculum/challenges/english/10-coding-interview-prep/rosetta-code/averages-root-mean-square.md [hunk 2]
```diff
@@ -8,11 +8,9 @@ dashedName: averagesroot-mean-square
 
 # --description--
 
-Compute the [Root mean square](<https://en.wikipedia.org/wiki/Root mean square> "wp: Root mean square") of the numbers 1 through 10 inclusive.
+Compute the Root mean square of the numbers 1 through 10 inclusive.
```
Comment (jeremylt) [inline curriculum/challenges/english/10-coding-interview-prep/rosetta-code/averages-root-mean-square.md:11]:
```suggestion
Compute the Root Mean Square (RMS) of the numbers 1 through 10 inclusive.
```
I think its standard to define abbreviations like this in line before using them..

File: curriculum/challenges/english/10-coding-interview-prep/rosetta-code/averages-root-mean-square.md [hunk 3]
```diff
@@ -8,11 +8,9 @@ dashedName: averagesroot-mean-square
...
 
-The *root mean square* is also known by its initials RMS (or rms), and as the **quadratic mean**.
-
-The RMS is calculated as the mean of the squares of the numbers, square-rooted:
+The <abbr title="Root mean square">RMS</abbr> is calculated by taking the mean of the squares of the numbers, and then square-rooting that number:
```
Comment (jeremylt) [inline curriculum/challenges/english/10-coding-interview-prep/rosetta-code/averages-root-mean-square.md:13]: maybe?
```suggestion
The <abbr title="Root mean square">RMS</abbr> is calculated by taking the square root of the mean of the squares of the numbers, given by the equation:
```

**Review by moT01 — APPROVED** [review]
LGTM 🎉

**Review by Nirajn2311 — APPROVED** [review]

**Pull request merged**